app/models/app_group.rb

```diff
@@ -48,4 +48,8 @@ def available?
   def max_tps
     self.infrastructure.options['max_tps'].to_i
   end
+
+  def new_total_tps(app_name,new_tps)
```

Comment:
Can you just make it one param which is new_tps gap instead of using where.not?

Reply:
Modified this in https://github.com/BaritoLog/BaritoMarket/commit/808c3fbfcc0412133a56de0bcbc4959b218c69c9